internal/syncer/connector/dingtalk_ai_table_test.go

```
@@ -0,0 +1,182 @@
+	})
+	if err != nil {
+		t.Fatalf("NewDingTalkAITableConnector failed: %v", err)
+	}
+	now := time.Date(2026, 8, 20, 1, 2, 3, 0, time.UTC)
+	connector.now = func() time.Time { return now }
+
+	session, err := connector.OpenSync(context.Background(), SyncRequest{FromBeginning: true, WindowEnd: now.Add(time.Hour)})
+	if err != nil {
+		t.Fatalf("OpenSync failed: %v", err)
+	}
+	defer session.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `session.Close` is not checked

(errcheck)